src/includes/getting-started-config/javascript.ember.mdx

```diff
@@ -0,0 +1,49 @@
+
+This snippet includes automatic instrumentation to [monitor the performance](./configuration/ember-options/) of your application, which registers and configures the Tracing integration, including custom Ember instrumentation.
```

Comment:
Because this links to the PM considerations for Ember, I'd modify this slightly to
```suggestion
This snippet includes automatic instrumentation to monitor the performance of your application, which registers and configures the Tracing integration, including custom [Ember instrumentation](./configuration/ember-options/).
```